manager/traverse.go

```diff
@@ -193,15 +193,30 @@ func traverseGitFiles(repoPath string) ([]model.Template, []error, error) {
 				}
 			}
 			var rancherCompose string
+			var templateVersion string
 			for _, file := range version.Files {
+
 				if file.Name == "rancher-compose.yml" {
 					rancherCompose = file.Contents
 				}
+
+				if strings.HasPrefix(file.Name, "template-") && strings.HasSuffix(file.Name, ".yml") {
```

Comment:
Oh. I think I misinterpreted the `version` part and thought that needed to accept like a version. Lol. I'll change that now.